src/components/dropdowns/Dropdown.spec.jsx

```diff
@@ -63,4 +63,23 @@ describe('<Dropdown />', () => {
     component.update();
     expect(component.find('.sg-dropdown__items')).toHaveLength(0);
   });
+
+  it('should pass event and link to onSelectHanlder', () => {
```

Comment:
Could you change description to something like 'should call onSelectHanlder when link is clicked' ?

Reply:
fixed in 7a6cfdac